pkg/configuration/constants.go

```diff
@@ -22,6 +22,7 @@ const (
 	RAW_CMD_ARGS                    string = "internal_raw_cmd_args"
 	WEB_APP_URL                     string = "internal_snyk_app"
 	INPUT_DIRECTORY                 string = "targetDirectory"
+	UNNAMED_PARAMETER               string = INPUT_DIRECTORY
```

Comment:
**nitpick(non-blocking):** It may be useful to move the comment around positional arguments to this declaration to improve readability.

Reply:
I'd go further and fix this... it's more than a nit to me.